webservices/common/models/reports.py

```diff
@@ -49,11 +49,11 @@ class CommitteeReports(PdfMixin, BaseModel):
     refunded_other_political_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_OTHER_POLITICAL_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
-    total_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
+    total_contributions_period = db.Column('total_contributions_period', db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
```

Comment:
Out of curiosity, why have these labels been added when we don't seem to have any where else?  Are they necessary and if so, should we consider adding it elsewhere going forward/